src/core/qgsrunprocess.cpp

```diff
@@ -262,7 +262,12 @@ int QgsBlockingProcess::run( QgsFeedback *feedback )
       const QByteArray ba = p.readAllStandardError();
       mStderrHandler( ba );
     } );
+#ifdef Q_OS_WIN
+    p.setNativeArguments( mArguments.join( ' ' ) );
```

Comment:
See https://doc.qt.io/qt-5/qprocess.html#start
"Windows: The arguments are quoted and joined into a command line that is compatible with the CommandLineToArgvW() Windows function. For programs that have different command line quoting requirements, you need to use setNativeArguments(). One notable program that does not follow the CommandLineToArgvW() rules is cmd.exe and, by consequence, all batch scripts."